user/enterprise/prerequisites.md

```diff
@@ -40,6 +46,10 @@ To register for a 30 day trial please visit
 [our signup page](https://enterprise.travis-ci.com/signup) to receive a trial license. Please email [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com) for
 more information on pricing.
 
+### High Availability Mode
+
+If you're interested in using High Availability, please let [sales](mailto:sales@travis-ci.com) know so we can get your trial license configured. Check out the [HA docs](`/user/enterprise/high-availability/') for more information.
```

Comment:
> please let [sales](mailto:sales@travis-ci.com) know

to:

`please let [us](mailto:sales@travis-ci.com) know`

I'd rather just say `us` because I feel that for the user it's not important if they're talking to sales or support for this.